tests/functional/settings.js

```diff
@@ -96,29 +92,6 @@ define([
         .end();
     },
 
-    'sign in to desktop context, go to settings, no way to sign out': function () {
```

Comment:
Do you mean in this branch or in some other branch? I don't see it elsewhere in this PR or in `settings_common.js` in `master`. Am I being dim?